apps/x509.c

```diff
@@ -674,32 +670,24 @@ int x509_main(int argc, char **argv)
     }
 
     if (reqfile) {
-        EVP_PKEY *pkey;
-
         req = load_csr(infile, informat, "certificate request input");
         if (req == NULL)
             goto end;
 
         if ((pkey = X509_REQ_get0_pubkey(req)) == NULL) {
-            BIO_printf(bio_err, "Error unpacking public key\n");
+            BIO_printf(bio_err, "Error unpacking public key from CSR\n");
             goto end;
         }
         i = do_X509_REQ_verify(req, pkey, vfyopts);
-        if (i < 0) {
-            BIO_printf(bio_err,
-                       "Error while verifying certificate request self-signature\n");
-            goto end;
-        }
-        if (i == 0) {
-            BIO_printf(bio_err,
-                       "Certificate request self-signature did not match the contents\n");
+        if (i <= 0) {
+            BIO_printf(bio_err, i < 0
+                       ? "Error while verifying certificate request self-signature\n"
```

Comment:
The indentation here looks wrong.

Reply:
It is actually fine. This is according to the auto-formatting by the Emacs C mode,
and `util/check-format.pl` does not complain - it did complain about a single statement surrounded by braces, which I've removed.